doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -60,52 +60,52 @@ The issues in this section are related to :ref:`protocols`.
 Amazon Sidewalk
 ===============
 
-.. rst-class:: v2-8-0
+.. rst-class:: v2-9-0 v2-8-0
 
 KRKNWK-19647: nRF52 has high power consumption in sleep mode for sub-GHz sample
-  nRF52840 device experiences high current consumption in sleep mode with Bluetooth LE link when using sub-GHz Amazon libraries.
+  nRF52840 device experiences high current consumption in sleep mode with Bluetooth® LE link when using sub-GHz Amazon libraries.
   This issue does not occur with Bluetooth LE-only configurations.
 
   **Affected platforms:** nRF52840
 
-.. rst-class:: v2-8-0
+.. rst-class:: v2-9-0 v2-8-0
 
 KRKNWK-19647: nRF53 has high power consumption in sleep mode
   The nRF5340 device has high current consumption in sleep mode across all link configurations.
 
   **Affected platforms:** nRF5340
 
-.. rst-class:: v2-8-0
+.. rst-class:: v2-9-0 v2-8-0
 
 KRKNWK-19459: AES crypt error occurs after a factory reset
```

Comment:
@divipillai All Amazon Sidewalk known issues are now resolved except for this one (KRKNWK-19459)

Reply:
Removed v2-9-0 for all entries except for KRKNWK-19459.